client/input/input.go

```diff
@@ -87,23 +87,21 @@ func GetCheckPassword(prompt, prompt2 string, buf *bufio.Reader) (string, error)
 // "y", "Y", "yes", "YES", and "Yes" all count as confirmations.
 // If the input is not recognized, it returns false and a nil error.
 func GetConfirmation(prompt string, buf *bufio.Reader) (bool, error) {
-	for {
-		if inputIsTty() {
-			fmt.Print(fmt.Sprintf("%s [y/N]: ", prompt))
-		}
-
-		response, err := readLineFromBuf(buf)
-		if err != nil {
-			return false, err
-		}
-
-		response = strings.ToLower(strings.TrimSpace(response))
-		if response[0] == 'y' {
-			return true, nil
-		}
-
-		return false, nil
+	if inputIsTty() {
```

Comment:
Is this desired behavior? Seems legit but I wonder why we looped to begin with?

Reply:
The original design was to loop until user typed something yet info displayed to users was incorrect: although users were shown `do you want this? [Y/n]`, if they didn't type anything default was to keep prompting rather than taking Enter as `yes`.

The default is now rather defensive (unless you type yes, we take it as a `no`), thus the for loop was a cruft leftover of my old commits. As such, I believe it should be removed :-)